server/src/main/java/io/spine/server/BoundedContext.java

```diff
@@ -253,13 +255,19 @@ public TenantIndex getTenantIndex() {
      * <p>If the repository does not have a storage assigned, it will be initialized
      * using the {@code StorageFactory} associated with this bounded context.
      *
+     * <p>Checks whether there is a default state for entity or aggregate type of the repository.
```

Comment:
As long as aggregates are also entities, let's remove `or aggregate` from this sentence.
`of the repository` is also obvious, as repositories always manage entities. So please remove this portion as well.